xbmc/FileItem.h

```diff
@@ -499,6 +499,9 @@ class CFileItem :
    */
   std::string GetBaseMoviePath(bool useFolderNames) const;
 
+  // Return the start of the string if there is a 'Disc n' subdirectory in the movie path, 0 if not
```

Comment:
if you want to provide doxy, use correct syntax, please (see how it is done at other places in this header).

Reply:
Done, i think.